packages/taraxa-ui/src/components/NetworkMenu/NetworkMenu.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { CssBaseline, IconButton, Menu, MenuItem, ThemeProvider, Tooltip } from '@material-ui/core';
+import { MenuDots, Check } from '../Icons';
+import useStyles from './NetworkMenu.styles';
+import theme from '../theme';
+
+enum Network {
```

Comment:
If this enum will be used by other components, I would move it in an utils folder and enum.ts file and export it there. And I think we might use it somewhere else.

Reply:
This shouldn't actually be anywhere in the react codebase TBH. It should come from the Web3 provider, like `provider.getNetwork().name` for example, with Ethers.

Reply:
> If this enum will be used by other components, I would move it in an utils folder and enum.ts file and export it there. And I think we might use it somewhere else.
I avoid splitting code into utils files until they are used. 
IMO I think that NetworkMenu is only for the taraxa explorer and we can make it more flexible if you want.